docs/building.md

```diff
@@ -14,27 +36,21 @@ environment.
 If a Go development environment is setup, one can use `go get` to install the
 `registry` command from the current latest:
 
-```sh
-go get github.com/docker/distribution/cmd/registry
-```
+    go get github.com/docker/distribution/cmd/registry
```

Comment:
I think the correct command is `go get github.com/docker/distribution`

This fetches all of the source.  I'm not quite sure what adding cmd/registry to the end does.